server/app.js

```diff
@@ -4,6 +4,12 @@ const path = require('path');
 const morgan = require('morgan');
 const api = require('./routes/api');
 
+try {
+  Object.assign(process.env, require('./.env.js'));
```

Comment:
@KristyHSC Could you please share the .env.js file with us on slack?

Reply:
Also, could you add the .env.js file to the .gitignore? I don't see a change for that file in this pr

Reply:
I uninstall the gitstar-components, and add .env.js to .gitignore. :)